### ISSUE TITLE
Zammad fails to parse filename* in Content-Disposition header of attachment

### ISSUE THREAD

**Issue by martinvonwittich** [zammad/zammad#2458]
### Infos:

* Used Zammad version: 2.8.x, but it's also reproducible in develop
* Installation method (source, package, ..): docker-compose
* Operating system: Debian 9
* Database + version: PostgreSQL 9.6.11 (zammad-docker-compose)
* Elasticsearch version: 5.6.14 (zammad-docker-compose)
* Browser + version: Chromium 71

### Expected behavior:

* An attachment with the following Content-Disposition header should be displayed in Zammad as `Erklärung.txt`:

```
Content-Disposition: attachment; filename*=utf-8''Erkl%C3%A4rung.txt
```

See:

- https://tools.ietf.org/html/rfc6266#section-4.3
- https://tools.ietf.org/html/rfc5987

### Actual behavior:

* Zammad displays the attachment as `utf-8''Erklärung.txt`.

### Steps to reproduce the behavior:

* Attach a file with an UTF-8 filename to a mail, and use a MUA that uses the `filename*` parameter as described in RFC6266. Here is an example mail that triggers the bug:

```
Sender: Martin von Wittich <redacted@redacted>
Message-ID: <f43f065c222c18bd8adf2b6597e89024@redacted>
Date: Tue, 29 Jan 2019 22:42:09 +0100
Subject: UTF-8-Attachment-Test
From: Martin von Wittich <redacted@redacted>
To: zammad <redacted@redacted>
MIME-Version: 1.0
Content-Type: multipart/mixed;
 boundary="_=_swift_1548798129_e1966f90b1db46de031aa1a8234266a8_=_"
X-Mailer: IServ v3


--_=_swift_1548798129_e1966f90b1db46de031aa1a8234266a8_=_
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: quoted-printable

test

--_=_swift_1548798129_e1966f90b1db46de031aa1a8234266a8_=_
Content-Type: text/plain; name="=?utf-8?Q?Erkl=C3=A4rung=2Etxt?="
Content-Transfer-Encoding: base64
Content-Disposition: attachment; filename*=utf-8''Erkl%C3%A4rung.txt

SGVsbG8gV29ybGQK

--_=_swift_1548798129_e1966f90b1db46de031aa1a8234266a8_=_--
```

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by MrGeneration** [comment]
I'll check on that soon(TM) - I also think that this might be at least related to #2425 at some point.
We'll know more soon

**Comment by SummerStorm** [comment]
Fixed via 30d3ce2b88c9bbbbd6800f120a7fdaac5893e900.
